starboard/android/arm/toolchain/BUILD.gn

```diff
@@ -21,7 +21,6 @@ gcc_toolchain("target") {
   cc = "$prefix/armv7a-linux-androideabi${android_ndk_api_level}-clang"
   cxx = "$prefix/armv7a-linux-androideabi${android_ndk_api_level}-clang++"
   ld = cxx
-  ar = "$prefix/arm-linux-androideabi-readelf"
```

Comment:
Maybe we should use this readelf tool to avoid confusion in the future? wdyt?

Reply:
It the proposal to have:
readelf = "$prefix/arm-linux-androideabi-readelf"
ar = "$prefix/arm-linux-androideabi-ar"

Reply:
Not quite -

Currently we have `ar = "ar"` - see the assignment on the next line in the existing code - and we're not considering changing that.

I think Andrew is asking if we should have:
```
readelf = "$prefix/arm-linux-androideabi-readelf"  // or something similar
ar = "ar"
```

This PR currently keeps the existing behavior for `readelf`, which is to fallback to the "readelf" default in the `gcc_toolchain`.

Reply:
That sounds good to me.

Reply:
As discussed offline, we'll keep the status quo for now.